### DISCUSSION TITLE
OpenTTD 14 Title Screen Competition

### DISCUSSION THREAD

**Discussion by 2TallTyler** [OpenTTD/OpenTTD#12282] [Title Screen Competition] [answered]
Hello everyone, and welcome to the OpenTTD 14 title screen competition! This is your opportunity to design and help choose the savegame which plays behind the main menu of OpenTTD.

The rules for the competition are as follows:

## Schedule
* Submission deadline: March 11, 2024, at 23:59:00 UTC.
* Voting opens by March 18, 2024.
* Voting closes: March 25, 2024, at 23:59:00 UTC.

## Entry and voting
Entries were posted to the TT-Forums thread: https://www.tt-forums.net/viewtopic.php?t=91448

Voting will take place in this discussion thread, by upvoting one of the replies to the thread.

## Title screen guidelines
* This will be a first impression of the game for many new players, so create something which you feel summarizes the game. This is usually a busy, exciting scene showing off many modes of transport, but there are no rules about what must be shown.
* You can use camera control signs to show many views of your game. Remember to test the commands you write, and that the player's screen resolution or window size can affect what gets shown and does not get shown. Test on multiple different resolutions if possible. Instructions on using the camera controls is [here](https://www.tt-forums.net/viewtopic.php?p=1248166#p1248166).
* Note that any signs you place will be visible. This excludes signs used for the camera control, which are removed when the game is loaded.
* Breakdowns will automatically be disabled. Cheating for money is OK. Unprofitable companies are OK. Building with multiple companies is OK. Building the game in the Scenario Editor is OK.
* Do not use AIs, GameScripts, or NewGRFs. These are automatically disabled.
* The game must be created in a clean, master branch version of OpenTTD or its official binaries. The game cannot be created in custom modified versions of OpenTTD or in popular patch packs such as JGR Patch Pack, LtJoker's Patch Pack, Reddit client, CityMania client, etc.

Feel free to check out [the OpenTTD 13 competition](https://github.com/OpenTTD/OpenTTD/discussions/10210) for inspiration.

**Comment by 2TallTyler** [comment]
## Entry 1: accipiter2000
Video: https://www.youtube.com/watch?v=ZmcwwEcIJmE
Save file: [accipiter2000.zip](https://github.com/OpenTTD/OpenTTD/files/14574173/accipiter2000.zip)

**Reply by PeterN** [reply]
IMHO this is disqualified because it used NewGRFs at one point.

**Comment by 2TallTyler** [comment]
## Entry 2: ErorrTNT
Video: https://www.youtube.com/watch?v=rg5nc5UkYus
Save file: [ErorrTNT.zip](https://github.com/OpenTTD/OpenTTD/files/14574184/ErorrTNT.zip)

**Reply by ErorrTNTcz** [reply]
I don't think the video gives justice to my title screen. For the best experience play on 1920x1080.

**Reply by henrygoldss** [reply]
Don't take this vote too seriously. In fact, I think it would be better without any voting at all.

**Comment by 2TallTyler** [comment, marked as answer]
## Entry 3: Plus C
Video: https://www.youtube.com/watch?v=t_IKCMq5F-M
Save file: [Plus C.zip](https://github.com/OpenTTD/OpenTTD/files/14574187/Plus.C.zip)

**Comment by 2TallTyler** [comment]
## Entry 4: 2TallTyler
Video: https://www.youtube.com/watch?v=XSLfUUvpwmo
Save file: [2TallTyler.zip](https://github.com/OpenTTD/OpenTTD/files/14574191/2TallTyler.zip)

**Comment by D1nz3y** [comment]
Why no one add some accidents in start screen...

**Comment by DefinitelyNotRau117** [comment]
There is a small suggestion: use video with sound in the preview.
It would seem like an insignificant detail, but it still greatly affects the perception of the menu. A noisy side-effects is not a very pleasant thing, t definitely distracts from, for example, thoughtfully reading the description of the settings.

Of course, you can download the save and listen to it yourself, but this is not nearly as convenient as having the sound “here and now”, directly on the video.

**Reply by ErorrTNTcz** [reply]
I second this. I tried to make my title screen loop when the music ends. Also making the game run in 1920x1080 in the videos would help, since everyone I know plays on that resolution.

**Reply by Loafdude** [reply]
I also did not find the videos very helpful. Who plays at those resolutions?

**Comment by Loafdude** [comment]
Not sure if this has been mentioned but the current v13 title screen is not large enough at 4k resolutions.
(You see the black edge of the map). 4K monitors are becoming common now.
Is there an easy way to swap the title map out? I can check the entries if needed

**Reply by DefinitelyNotRau117** [reply]
![изображение](https://github.com/OpenTTD/OpenTTD/assets/115672237/62336943-0a83-4091-826e-5e984c2b5cd2)
Inside the archive there is a save and a `opntitle.dat`
You can view the save just like any other save, but to load it onto titlegame you need to drop with replace `opntitle.dat` file into the `baseset` folder in the game directory (where you have openttd.exe)

**Comment by PeterN** [comment]
> Not sure if this has been mentioned but the current v13 title screen is not large enough at 4k resolutions. (You see the black edge of the map). 4K monitors are becoming common now.

Assumption is that on a 4K monitor you are using 2x or even 4x interface scale, which mitigates that somewhat.

**Reply by TheMowgliMan** [reply]
Not neccesarily; on a larger screen 1x would be a similar size as on a standard size 1080p monitor. But answering questions like this is why yall added the survey.

**Reply by 2TallTyler** [reply]
For what it's worth, I recorded the videos on a 1080p monitor with 2x interface scale: my personal setup for playing OpenTTD.

**Reply by TheMowgliMan** [reply]
> For what it's worth, I recorded the videos on a 1080p monitor with 2x interface scale: my personal setup for playing OpenTTD.

True. But I suspect many people who started playing before the interface scale updates (like me) play with 1x, or play that because they don't know about the other options. I personally don't like the appearance of anything other than 1x, and if people have the mouse accuracy required for 1x on higher resolutions, I suspect they use that.

But again, this is a question for the survey.
